.gitignore

```diff
@@ -2,4 +2,5 @@
 *.ipr
 *.iws
 target
+/.idea
```

Comment:
.idea should be enough https://github.com/github/gitignore/blob/master/Global/JetBrains.gitignore#L6